docs/contributing/code.md

```diff
@@ -32,17 +32,17 @@ go mod download
 
 Finally, we use [direnv](https://direnv.net/) to manage env vars in development. This is primarily used for running tests. You can either:
 
-- Set up your own by using `.envrc.example` as a base. You're unlikely to need to set anything other than `GITHUB_WORKSPACE` for integration tests, so feel free to skip this step or export it manually using: ```export GITHUB_WORKSPACE=`pwd` ```.
+- Set up your own by using `.envrc.example` as a base. You're unlikely to need to set anything other than `GITHUB_WORKSPACE` for integration tests, so feel free to skip this step or export it manually using: ``export GITHUB_WORKSPACE=`pwd` ``.
```

Comment:
Is this correct? From three to two (not one) back ticks?

Reply:
seems good til you can do `` if you need to escape ` :)

Reply:
Niiiice 😎